migrations/5_deploy_parameterizer.js

```diff
@@ -49,7 +49,7 @@ module.exports = (deployer, network, accounts) => {
     );
   })
     .then(async () => {
-      if (network === 'test' || network === 'coverage') {
+      if (network === 'development' || network === 'test' || network === 'coverage') {
```

Comment:
also remove this, if we can. I think we added `development` config, but the original tcr repo doesn't have this